sig-architecture/production-readiness.md

```diff
@@ -5,21 +5,90 @@ Kubernetes are observable, scalable and supportable, can be safely operated in
 production environments, and can be disabled or rolled back in the event they
 cause increased failures in production.
 
-More details may be found in the [PRR KEP].
+Production readiness reviews are done by a separate team, apart from the SIG
+leads (although SIG lead approval is needed as well, of course). It is useful to
+have the viewpoint of a team that is not as familiar with the intimate details
+of the SIG, but *is* familiar with Kubernetes and with operating Kubernetes in
+production. Experience through our dry runs in 1.19 and 1.20 have shown that
+this slightly "outsider" view helps identify otherwise missed items.
 
-## Status
+More background may be found in the [PRR KEP].
 
-As of 1.19, production readiness reviews are required, and are part of the KEP
-process. The PRR questionnaire previously found here has been incorporated into
-the [KEP template]. The template details the specific questions that must be
-answered, depending on the stage of the feature.
+## Status
 
 As of 1.21, PRRs are now blocking. PRR _approval_ is required for the enhancement
-to be part of the release.
+to be part of the release. This means that any KEPs targeting the release for any
+stage, and in - or moving to - `implementable` state, will require production
```

Comment:
That is potentially something we want to discuss. While targeting release should definitely imply PRR approval, I'm not convinced about moving to implementable.
I've seen multiple cases where KEPs moved to implementable was staying there for quite some time with not much traction so I'm not sure we want to necessary (a) all load to PRR team, (b) block authors from having that merged...

Reply:
it's "target release && implementable", rather than just "target release". It's actually mandatory to move to "implementable" to be in the release, per the Enhancements team (see the KEP template).

Without checking for "implementable", KEPs that are still in "provisional" status can sometimes run into the CI validation.

Reply:
So given the fact that you need to be "implementable" to target a release - do we need to mention this explicitly?

Reply:
No, good point.